Design-System/styles/form.css

```diff
@@ -0,0 +1,27 @@
+/*
+    Author: Vrushabh Dhatrak
+    Release: 0.1
+    Description : Form CSS
+*/
+.form_section {
+    display: flex;
+    flex-direction: column;
+    height: 90vh;
+    
+}
+.form {
+    margin:5rem 0 0;
+    position: relative;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.form-btn{
+    width: 53.6rem !important;
+    max-width:none !important;
+    margin:0 !important;
```

Comment:
why used important

Reply:
Yes it's important because I want to override some component library properties on that class